### DISCUSSION TITLE
I want to obtain badges thanks for answer

### DISCUSSION THREAD

**Discussion by rov1dev** [community/community#142465] [Other Feature Feedback, Questions, & Ideas]
It helps me to get job

**Comment by webdevmahdi** [comment]
Mark my comment as an answer. This will also help me.

**Comment by ibrokhimovme** [comment]
salomlar

**Reply by rov1dev** [reply]
aliklar

**Reply by rov1dev** [reply]
asdas

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142467#discussioncomment-11042195) for reference to why this discussion was closed and locked. Thanks!
